Tests/API/e2e/KarateTests.feature

```diff
@@ -0,0 +1,78 @@
+﻿Feature: Test Flow
```

Comment:
The tests are not actual right now. I think we can remove them in this branch, and do it in the next one